### PR TITLE
Fix various minor spelling mistakes

### PR DESCRIPTION
**Pull request by adrianwright109**
Fixes minor spelling mistakes in Components area.

### CODE REVIEW

**Review by javiercn — APPROVED** [review]

File: src/Components/Components/test/ParameterViewTest.Assignment.cs [hunk 1]
```diff
@@ -71,7 +71,7 @@ public void IncomingParameterMatchesInheritedDeclaredParameter_SetsValue()
         }
 
         [Fact]
-        public void IncomingParameterMatchesOverridenParameter_ThatDoesNotHasAttribute()
+        public void IncomingParameterMatchesOverriddenParameter_ThatDoesNotHasAttribute()
```
Comment (SteveSandersonMS) [inline src/Components/Components/test/ParameterViewTest.Assignment.cs:74]:
```suggestion
        public void IncomingParameterMatchesOverriddenParameter_ThatDoesNotHaveAttribute()
```

**Review by pranavkm — CHANGES_REQUESTED** [review]
@adrianwright109 could you undo changes to the submodule?

**Comment by adrianwright109** [comment]
Changes to submodule have been undone.

**Review by captainsafia — APPROVED** [review]
Coolio! Thanks for submitting this contribution and addressing the follow-ups.

**Comment by captainsafia** [comment]
@pranavkm Can you re-review you this? Your review is blocking the merge.

**Review by pranavkm — APPROVED** [review]

**Comment by pranavkm** [comment]
Thanks for the PR!

**Pull request merged**